builtin/providers/aws/resource_aws_subnet.go

```diff
@@ -108,35 +111,35 @@ func resourceAwsSubnetRead(d *schema.ResourceData, meta interface{}) error {
 
 	subnet := &resp.Subnets[0]
 
-	d.Set("vpc_id", subnet.VpcId)
+	d.Set("vpc_id", subnet.VPCID)
 	d.Set("availability_zone", subnet.AvailabilityZone)
-	d.Set("cidr_block", subnet.CidrBlock)
-	d.Set("map_public_ip_on_launch", subnet.MapPublicIpOnLaunch)
-	d.Set("tags", tagsToMap(subnet.Tags))
+	d.Set("cidr_block", subnet.CIDRBlock)
+	d.Set("map_public_ip_on_launch", subnet.MapPublicIPOnLaunch)
+	d.Set("tags", tagsToMapSDK(subnet.Tags))
```

Comment:
@phinze  I didn't dereference these, but acceptance tests pass :/ 

Is there some magic in `d.Set` that's setting the appropriate value? I started digging but ended up in the `reflect` package and was a bit over my head. 

If not, and I'm doing something wrong here, do you have an example of an acceptance test that I can throw in to `subnet_test.go` to catch this kind of thing? /cc @mitchellh